packages/bigtable/src/table.js

```diff
@@ -624,9 +624,27 @@ Table.prototype.getRows = function(options, callback) {
  *     See {module:bigtable/table#mutate}.
  * @param {function} callback - The callback function.
  * @param {?error} callback.err - An error returned while making this request.
- * @param {object} callback.apiResponse - The full API response.
+ * @param {object} callback.insertErrors - A status object for each failed
+ *     insert.
  *
  * @example
+ * var callback = function(err, insertErrors) {
+ *   if (err) {
+ *     // Error handling omitted.
+ *   }
+ *
+ *   // insertErrors = [
+ *   //   {
+ *   //     code: 500,
+ *   //     message: 'Internal Server Error',
+ *   //     entry: {
```

Comment:
This entry should be the same entry the user gave us. In this example case, it would be `{ key: 'alincoln', ... }`